delta/plugins/storage/src/test/scala/ch/epfl/bluebrain/nexus/delta/plugins/storage/files/FilesSpec.scala

```diff
@@ -161,6 +163,18 @@ class FilesSpec(docker: RemoteStorageDocker)
         actual shouldEqual expected
       }
 
+      "succeed when the file has special characters" in {
+        val specialFileName = "-._~:?#[ ]@!$&'()*,;="
```

Comment:
Note that:
1. I believe there is still a problem if a filename contains `%`
2. If a `+` is in the filename, it will end up being a space when saved